lib/galaxy/dependencies/conditional-requirements.txt

```
@@ -14,7 +14,7 @@ python-pam
 galaxycloudrunner
 
 # For file sources plugins
-fs.webdavfs  # type: webdav
+fs.webdavfs==0.4.2  # type: webdav
```

Maybe a minimum version is enough?
```suggestion
fs.webdavfs>=0.4.2  # type: webdav
```

---

Sure!